.circleci/config.yml

```diff
@@ -6,31 +6,28 @@ jobs:
   install_deps:
     executor:
       name: node/default
-      tag: "14.16.1"
+      tag: "14.16.0"
```

Comment:
Although this is a slight downgrade, it brings this template into parity with spraygun-react (JS), which is at 14.16.0. For maintainability I decided it was easier if both projects use the same Circle CI config.